core/types/market.go

```diff
@@ -637,6 +639,8 @@ type Market struct {
 func MarketFromProto(mkt *proto.Market) (*Market, error) {
 	asset, _ := mkt.GetAsset()
 	lppr, _ := num.DecimalFromString(mkt.LpPriceRange)
+	linearSlippageFactor, _ := num.DecimalFromString(mkt.LinearSlippageFactor)
+	quadraticSlippageFactor, _ := num.DecimalFromString(mkt.QuadraticSlippageFactor)
```

Comment:
Shouldn't we handle these errors?

Reply:
no. it's not possible for this error to occur here.